client/src/utils/deleteTokenOn401StatusCodes.js

```diff
@@ -0,0 +1,19 @@
+export default (axios) => {
+  const successHandler = (response) => {
+    return response;
+  };
```

Comment:
wat

Reply:
if you really must have this function, then you can do `const successHandler = (response) => response`

Reply:
Fixed.